lib/hubspot/company.rb

```diff
@@ -40,6 +40,32 @@ def all(opts={})
         response['results'].map { |c| new(c) }
       end
 
+      # Find all companies by created date (descending)
+      #    recently_updated [boolean] (for querying all accounts by modified time)
+      #    count [Integer] for pagination
+      #    offset [Integer] for pagination
+      # {http://developers.hubspot.com/docs/methods/companies/get_companies_created}
+      # {http://developers.hubspot.com/docs/methods/companies/get_companies_modified}
+      # @return [Object], you can get:
+      # response.results for [Array]
+      # response.hasMore for [Boolean]
+      # response.offset for [Integer]
+      def all_with_offset(opts = {})
+        recently_updated = opts.delete(:recently_updated) { false }
+
+        path = if recently_updated
+          RECENTLY_MODIFIED_COMPANIES_PATH
+        else
+          RECENTLY_CREATED_COMPANIES_PATH
+        end
+
+        response = Hubspot::Connection.get_json(path, opts)
+        formatted_results = response['results'].map { |c| new(c) }
+        response = JSON.parse(response.to_json, object_class: OpenStruct)
```

Comment:
I like the idea of using `OpenStruct` here to make a cleaner interface for the object attributes, but all of the other API calls return a hash. In the next version we can definitely take a look at updating all of the interfaces to do something like this but for this version, we should continue to match the existing conventions.

Check out `Deal#all` for an example of how this is currently being handled.

Reply:
Good, modified 👍

Reply:
btw, if there's any certain plan for next version, feel free to notify me, let me see if I can help something :)
Thanks.